charts/durabletask-azurestorage-external-scaler/templates/rolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
`ClusterRoleBinding` instead so that could access objects in different namespaces.